packages/api/endpoints/collections.js

```diff
@@ -176,7 +166,6 @@ async function put(req, res) {
   const {
     collectionPgModel = new CollectionPgModel(),
     knex = await getKnexClient(),
-    esClient = await getEsClient(),
     collectionConfigStore = new CollectionConfigStore(
       process.env.system_bucket,
```

Comment:
this is slight scope creep so *nit* but
getRequiredEnvVar('system_bucket', process.env) here, and elsewhere in this file
(107, 108, 171, 172, 228, 229) would shift us towards better typing while you're in this file

Reply:
yes exactly